index.html

```diff
@@ -155,7 +155,8 @@ <h3>{{RTCPeerConnection/addTransceiver()}}</h3>
            {{RTCRtpTransceiverInit/sendEncodings}} within
            {{RTCPeerConnection/addTransceiver()}}. To validate
            {{RTCRtpEncodingParameters/scalabilityMode}}, add
-           the following validation steps after step 3 (within step 8):
+           the following steps to the
+           [=RTCPeerConnection/addTransceiver sendEncodings validation steps=]:
            <ol>
              <li>
                If <var>sendEncodings</var> contains any encoding whose
```

Comment:
While you're here, maybe replace the uses of `RTCRtpSender.getCapabilities` below with the new algos? (github won't let me drive-by comment outside of the patch)
```diff
-               {{RTCRtpSender.getCapabilities(kind)}}.<code>codecs</code>,
+               [=list of implemented send codecs=] for <var>kind</var>.,
```
Or were they not exported?